### PR TITLE
Do not always include LUTS

### PR DESCRIPTION
**Pull request by kaneparkinson**
LUTS are never excluded by filters, so setting a filter like `--filter AnalysisType=policy,rule` will still include LUTS. Also, the LUTS file paths are always included as part of `search_directories` in `test_analysis`.

This change removes them from the list of always included paths for `test_analysis` and prevents filter_analysis from always returning `True` when dir_name matches `*lookup_tables*`

### CODE REVIEW

**Comment by michael-cariello** [comment]
Looks good to me , if you can please sign the CLA I can go ahead and approve and thank you for your contribution!

**Comment by kaneparkinson** [comment]
> 

Thanks. I signed the CLA and clicked recheck but it hasn't updated the status yet. Does someone need to approve workflows for CLA assistant to re-run?

**Comment by michael-cariello** [comment]
I do not have access to the CLA assistant and I am unsure why recheck is not working. I have opened a ticket with our IT and will get back to you shortly.

**Review by michael-cariello — APPROVED** [review]

**Comment by kaneparkinson** [comment]
> I do not have access to the CLA assistant and I am unsure why recheck is not working. I have opened a ticket with our IT and will get back to you shortly.

Thanks. Should I also bump version before merging?

**Comment by michael-cariello** [comment]
> > I do not have access to the CLA assistant and I am unsure why recheck is not working. I have opened a ticket with our IT and will get back to you shortly.
> 
> Thanks. Should I also bump version before merging?

That would be great thank you !

**Review by michael-cariello — APPROVED** [review]

**Comment by kaneparkinson** [comment]
Ready to merge. (I don't have write access)

**Pull request merged**
